tests/TestCase/Database/Type/DateTimeTypeTest.php

```diff
@@ -212,7 +212,6 @@ public function marshalProvider()
             ['', null],
             ['derpy', null],
             ['2013-nope!', null],
-            ['13-06-26', null],
```

Comment:
2-digit years are valid in DateTime.  It will produce a year offset from 0 with the 'Y' formatter: '0013'